ServerAdministration/PractiaclDocker.md

```diff
@@ -0,0 +1,139 @@
+
+```bash
+docker swarm ls
+```
+
+The output should look like this:
+
+| ID                           | HOSTNAME                   | STATUS             | AVAILABILITY       | MANAGER STATUS     | ENGINE VERSION |
+| :--------------------------- | :------------------------- | :----------------- | :----------------- | :----------------- | :------------- |
+| yny9ky6b6zczqrjzxd7sl71k6 *  | giraf-master00.srv.aau.dk  | Ready              | Active             | Leader             | 18.09.3        |
+| 2n08r588w9p8xazc5cm8r6o9o    | giraf-master01.srv.aau.dk  | Ready              | Active             | Reachable          | 18.09.3        |
+| wrr68nqt116tk1rszwvdv1nmk    | giraf-node00.srv.aau.dk    | Ready              | Active             |                    | 18.09.3        |
+| bhh5mitvwzdhzbky1cjne9ffg    | giraf-node01.srv.aau.dk    | Ready              | Active             |                    | 18.09.3        |
+| as7n375y2gwcj5vf4h73h9ron    | giraf-node02.srv.aau.dk    | Ready              | Active             |                    | 18.09.3        |
+| koclcs8nxt0y6qu4ho511la0m    | giraf-node03.srv.aau.dk    | Ready              | Active             |                    | 18.09.3        |
+
+
+## Examples of using a service
+
+Once the Docker Swarm has been set up it can be used to serve the different parts of the project. In the following we will give some examples on how to work with the Swarm.
+
+### Example nginx
+
+To create a new service use the following:
+
+```bash
+docker service create --name nginx-giraf-proxy --replicas 2 -p 80:80 -p 443:443 nginx:1.15
+```
+
+Be running this command an service with two containers will be stated and the containers will have port 80 and 443 exposed to the internet and can be accessed on the Swarms public IP's. The two containers will be running the nginx version 1.15.
+
+To upgrade the version that a service uses run the following to downgrade or upgrade the service:
+
+```bash
+docker service update --image nginx:1.12 nginx-giraf-proxy
+```
+
+To encrease the number of contaners a service creates use the following command:
+
+```bash
+docker service scale nginx-giraf-proxy=5
+```
+
+To verify the service use the following command:
+
+```bash
+docker service ps nginx-giraf-proxy
+```
+
+The outpus should look like this:
+
+| ID                 | NAME               | IMAGE              | NODE                       | DESIRED STATE      | CURRENT STATE           | ERROR              | PORTS |
+| :------------------|:-------------------|:-------------------|:---------------------------|:-------------------|:------------------------|:-------------------|:-----|
+| iksklc50ttxt       | nginx-giraf-proxy.1| nginx:latest       | giraf-master01.srv.aau.dk  | Running            | Running 29 minutes ago  |                    ||
+
+Note that the ports exposed is not listed in this view since it is served throug the service and can be seen then running the following command:
```

Comment:
```suggestion
Note that the ports exposed are not listed in this view since it is served through the service and can be seen then running the following command:
```